### ISSUE TITLE
Too many scripts corresponding to README.md were not provided in braker3

### ISSUE THREAD

**Issue by oddguyeee** [Gaius-Augustus/BRAKER#901]
![image](https://github.com/user-attachments/assets/cec5f22f-a4a3-4549-959f-225ffb34b5bd)
![image](https://github.com/user-attachments/assets/33f77965-4a58-4a3b-aa47-780b9484aa04)
![image](https://github.com/user-attachments/assets/10155470-3a1c-49db-9370-f316bfe1a4ed)

**Comment by KatharinaHoff** [comment]
Thank you for pointing out the outdated documentation!

I removed align2hints.pl and startAlign.pl from the README.md file since they are not longer required to run BRAKER. They are a relict of times prior GeneMark-EP and GeneMark-ETP integration. Fixed with https://github.com/Gaius-Augustus/BRAKER/commit/075383e04b0f399cae64c17d5f557746b6de3095 .

The script change_path_in_perl_scripts.pl is also no longer required since the GeneMark team changed the shebang lines of those perl scripts to /usr/bin/env python3. I updated documentation for this in commit https://github.com/Gaius-Augustus/BRAKER/commit/ece9e0f28ea2634b5abbfd0b7cf0a6be5f1bf8db . 

If for historical reasons you still want to find the scripts align2hints.pl and startAlign, they are still available in GALBA. I disentangled BRAKER and GALBA a while back because it was controversial to include tools other than GeneMark in BRAKER. https://github.com/Gaius-Augustus/GALBA/blob/main/scripts/startAlign.pl https://github.com/Gaius-Augustus/GALBA/blob/main/scripts/aln2hints.pl (name of second script changes, but it has the same functionality, slightly expanded).
